src/binary-encoding.adoc

```diff
@@ -202,13 +205,11 @@ pairs are spliced into the parent struct (TODO: Link)
 ----
 
 [[opcodes]]
-== Opcodes
+=== Opcode Overview
```

Comment:
I don't feel strongly about it, but because this defines the term "opcode" for the first time, I'd leave the title as "Opcodes."